src/main/java/Events.java

```diff
@@ -0,0 +1,46 @@
+import java.text.ParseException;
```

Comment:
I think it's better to set the class name as Event rather than Events because normally plural form implies that the object/class is a collection.